browser/dasher/messageStore.js

```diff
@@ -27,8 +28,12 @@ export default class MessageStore {
 
     viewMessageStore(){
       this._load_from_browser().catch(() => {}).then(messages => {
```

Comment:
Maybe add a comment near here that says what `messages` is. Looks like it is an array of arrays of string?

Reply:
Somewhat better, looks like a map was better for this case